pkgs/applications/audio/ardour/default.nix

```diff
@@ -156,16 +156,16 @@ stdenv.mkDerivation rec {
     # wscript does not install these for some reason
     install -vDm 644 "build/gtk2_ardour/ardour.xml" \
       -t "$out/share/mime/packages"
-    install -vDm 644 "build/gtk2_ardour/ardour6.desktop" \
+    install -vDm 644 "build/gtk2_ardour/ardour7.desktop" \
```

Comment:
```suggestion
    install -vDm 644 "build/gtk2_ardour/ardour${lib.versions.major version}.desktop" \
```
and for the others

Reply:
Might be better make a common derivation with these parameters, so none of this is duplicated -- assuming the build process etc stays the same.

Reply:
@winterqt I'm the person that suggested having 2 Ardour versions at the same time and as far as I'm concerned, we only need Ardour6 for a limited amount of time, say a few months to max a year.
So if it's too much of a hassle to deduplicate the code, I'd say: don't bother.

Reply:
I've applied this suggestion to `default.nix` instead, which should help make updating ardour more trivial in the future.

As @magnetophon notes, I've left `6.nix` alone as it's likely to be removed in the not-too-distant future.